.github/workflows/test.yml

```diff
@@ -44,8 +51,12 @@ jobs:
 
       - name: Start Pi-hole
         shell: bash
-        run: |-
-          docker-compose -f docker-compose.yml -f <(echo '{"services": {"pihole":{"image": "pihole/pihole:${{ matrix.tag }}"}}}') up -d --build
+        env:
+          TAG: ${{ matrix.tag }}
+        run: make docker-run
```

Comment:
I don't see how the image is being set with the new flow. When I run it I get the latest version I have pulled.

```
➜  terraform-provider-pihole git:(main) ✗ ENV=nightly docker compose up -d --build
[+] Building 0.0s (0/0)                                                                                                                                                                          docker:desktop-linux
[+] Running 1/1
 ✔ Container pihole  Started                                                                                                                                                                                     0.0s
➜  terraform-provider-pihole git:(main) ✗ docker ps
CONTAINER ID   IMAGE                   COMMAND      CREATED         STATUS                            PORTS                                          NAMES
d834d406580e   pihole/pihole:2022.05   "/s6-init"   4 seconds ago   Up 3 seconds (health: starting)   53/udp, 53/tcp, 67/udp, 0.0.0.0:8080->80/tcp   pihole
```

Reply:
Hmm, when I run the same command as you, my scripts fails. I'm running it on Fedora linux with docker `Docker version 26.1.4, build 5650f9b`.

```
❯ ENV=nightly docker compose up -d --build
WARN[0000] The "TAG" variable is not set. Defaulting to a blank string. 
unable to get image 'pihole/pihole:': Error response from daemon: invalid reference format
```

However, if you change the variable `ENV` to `TAG` it might yield better results. Does that make any difference?

Reply:
ah my bad, clearly its supposed to be TAG. I  missed that it was referenced in the compose file itself